config/locales/fr.yml

```diff
@@ -212,7 +226,13 @@ fr:
     signature: Notifications de Mastodon depuis %{instance}
     view: 'Voir :'
   applications:
+    created: Application créée avec succès
+    destroyed: Application supprimée avec succès
     invalid_url: L’URL fournie est invalide
+    regenerate_token: Regénérer le jeton d'accès
+    token_regenerated: Jeton d'accès regénéré avec succès
+    warning: Soyez prudent⋅e avec ces données. Ne les partagez pas!
```

Comment:
Typo: in french, there is a space before (and after) `!`

Reply:
More precisely, in French the exclamation mark requires a thin non breakable space ('espace fine') before and a plain (breakable) space after (as do interrogation point, colon, semicolon and some others).